packages/kbn-grid-layout/grid/use_grid_layout_events.ts

```diff
@@ -180,7 +180,7 @@ export const useGridLayoutEvents = ({
       const atTheTop = window.scrollY <= 0;
       const atTheBottom = window.innerHeight + window.scrollY >= document.body.scrollHeight;
 
-      const startScrollingUp = !isResize && heightPercentage < 5 && !atTheTop; // don't scroll up when resizing
+      const startScrollingUp = heightPercentage < 5 && !atTheTop;
```

Comment:
Talked with @ThomThomson and we decided to allow scrolling up when resizing because it's useful when panels are taller than the viewport. You almost always end up with a panel that takes up only a single row when doing this though, which is why I disabled it originally 🙈